frontend/src/pages/devices/List.tsx

```diff
@@ -153,6 +155,16 @@ export const List = () => {
         return <>{ level }%</>;
     };
 
+    const renderOnline = (entity: DeviceT) => {
+        if (entity.rawDevice.status === "ONLINE") {
+            return <Badge variant="success">Online!</Badge>;
```

Comment:
Thoughts on:
* changing the text to `Online`
* changing the badge to something without color - "online" should be the expected state, so I don't think we should call it out (know this isn't what was requested, but I think we should try this out and see what they think)